### PR TITLE
CMS 407 updates sizes

### PR DESCRIPTION
**Pull request by lory87**
**This pull request fixes the following issues**:

- https://divio-ch.atlassian.net/browse/CMS-407

Design:
00_guide-django-cms

Before:
screen shot 2015-09-21 at 17 11 24

After:
screen shot 2015-09-21 at 17 11 06

Before:
screen shot 2015-09-21 at 17 11 31

After:
screen shot 2015-09-21 at 17 11 42

### CODE REVIEW

File: djangocms_admin_style/sass/settings/_cms.scss [hunk 1]
```diff
@@ -7,7 +7,7 @@ $speed-base: 200ms;
 $white: #fff;
 $black: #000;
 
-$color-primary: #57bcf0;
+$color-primary: #00bbff;
```
Comment (Brankub) [inline djangocms_admin_style/sass/settings/_cms.scss]: wow, this could change everything :)

**Comment by Brankub** [comment]
Looks great, but let's wait for @vxsx review.

File: djangocms_admin_style/sass/settings/_cms.scss (hunk 1 above)
Comment (lory87) [inline djangocms_admin_style/sass/settings/_cms.scss]: yep have to change the divio blue color :)

**Comment by vxsx** [comment]
:+1:

**Pull request merged**